src/viper/trans.ml

```diff
@@ -129,9 +123,9 @@ and unit' (u : M.comp_unit) : prog =
     let is' = List.map (fun mk_i -> mk_i ctxt') mk_is in
     (* given is', compute ghost_is *)
     let ghost_is = List.map (fun mk_i -> mk_i ctxt') !(ctxt.ghost_items) in
-    let init_id = id_at "__init__" Source.no_region in
-    let self_id = id_at "$Self" Source.no_region in
-    let self_typ = { it = RefT; at = self_id.at; note = NoInfo } in
+    let init_id = !!! (Source.no_region) "__init__" in
```

Comment:
I think `id_at` can go now :-)

Reply:
Yeah, that was actually my intention but I failed to push it through. Now done.